<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RNGForm</class>
 <widget class="QWidget" name="RNGForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор случайных вариантов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>170</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>638</width>
         <height>168</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>50</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>вариантов:</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBoxFormula">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>349</width>
          <height>145</height>
         </rect>
        </property>
        <property name="title">
         <string>Формула лотереи</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Основные номера</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>сколько:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxNumberOfBalls1">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999999</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>от:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxStartOfBalls1">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>9999999</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>до:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxEndOfBalls1">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999999</number>
              </property>
              <property name="value">
               <number>36</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Дополнительные номера</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>сколько:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxNumberOfBalls2">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>9999999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>от:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxStartOfBalls2">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>9999999</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>до:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxEndOfBalls2">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999999</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButtonGenerate">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>20</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Генерировать</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxVariantsCount">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>70</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxSort">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>110</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>sorting in ascending order</string>
        </property>
        <property name="text">
         <string>по возрастанию</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
